.storybook/main.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  stories: ['../**/*.stories.js'],
```

Comment:
Could probably make the regex match `index.stories.js`, but keeping this for now in case we want to break from that convention.